lib/Db/ShareWrapperRequest.php

```diff
@@ -360,14 +360,16 @@ public function getSharesBy(
 	 * @param FederatedUser $federatedUser
 	 * @param int $nodeId
 	 * @param bool $reshares
+	 * @param bool $shallow Whether the method should stop at the first level, or look into sub-folders.
 	 *
 	 * @return ShareWrapper[]
 	 * @throws RequestBuilderException
 	 */
 	public function getSharesInFolder(
 		FederatedUser $federatedUser,
 		int $nodeId,
-		bool $reshares
+		bool $reshares,
+		$shallow = true
```

Comment:
bool ?

Reply:
Yes, need a type if there is a default value ?

Reply:
I don't know, but I personally like when it is set, even with a default value :)